src/ADTypes.jl

```diff
@@ -43,6 +43,7 @@ export AutoChainRules,
        AutoGTPSA,
        AutoModelingToolkit,
        AutoMooncake,
+       AutoMooncakeForward,
```

Comment:
For completeness, I suggest we consider introducing `AutoMooncakeBackward`, then alias `AutoMooncake = AutoMooncakeBackward`.

Reply:
I think having two names for the same thing introduces unneeded confusion. We can remove `AutoMooncakeForward` at the next breaking release of ADTypes if there ever is one